gnosis-journal-of-gnostic-studies.csl

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
```

Comment:
The style guide allows either British or American English. Would it make sense to drop the `default-locale`?

Reply:
The only real parts where that shows is long dates where the day and month switches as well as how quotation marks are shown and handled.
This style has neither long dates (like e.g. for websites) nor quotation marks. So I don't think it matters too much in the case of this citation style and is more about the written text.
I'd leave as is and if someone comes requesting a change that's easy to make.